cmd/utils/utils.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import "os"
+
+func FileExists(filePath string) bool {
+	if _, err := os.Stat(filePath); err != nil {
+		return false
+	}
+	return true
+}
```

Comment:
Returned error can be permission related, not always not found.
```suggestion
func FileExists(filePath string) bool {
	_, err := os.Stat(filePath)
	return err == nil || !os.IsNotExist(err)
}
```